packs/feats/invoke-the-elements.json

```diff
@@ -32,7 +32,60 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "adjustName": false,
+                "choices": [
+                    {
+                        "label": "PF2E.TraitBludgeoning",
+                        "predicate": [
+                            "feat:brine-may"
+                        ],
+                        "value": "bludgeoning"
+                    },
+                    {
+                        "label": "PF2E.TraitCold",
+                        "predicate": [
+                            "feat:snow-may"
+                        ],
+                        "value": "cold"
+                    },
+                    {
+                        "label": "PF2E.TraitFire",
+                        "predicate": [
+                            "feat:veil-may"
+                        ],
+                        "value": "fire"
+                    },
+                    {
+                        "label": "PF2E.TraitElectricity",
+                        "predicate": [
+                            "feat:virga-may"
+                        ],
+                        "value": "electricity"
+                    }
+                ],
+                "flag": "changelingType",
+                "key": "ChoiceSet"
+            },
+            {
+                "key": "ActiveEffectLike",
+                "mode": "override",
+                "path": "flags.pf2e.changeling.changelingType",
+                "value": "{item|flags.pf2e.rulesSelections.changelingType}"
+            },
+            {
```

Comment:
Is this necessary?

Reply:
This is what adds/changes the damage type on the roll. Unless I'm overthinking this

Reply:
Let me test by removing it from the feat and see what happens

Reply:
Yup - it's overkill. It works without it.
I'll delete it now